mmc/item_updater_mmc.cpp

```diff
@@ -78,6 +78,9 @@ void ItemUpdaterMMC::reset()
     // Set attribute to clear hypervisor NVRAM
     utils::setClearNvram(bus);
 
+    // reset the enabled property of dimms/cpu after factory reset
```

Comment:
We are not allowing a processor to manually guard through the Redfish.
We are supporting DIMM and Core through the Redfish. Please change it.